src/gtk1/menu.cpp

```diff
@@ -978,10 +983,10 @@ bool wxMenu::GtkAppend(wxMenuItem *mitem, int pos)
     else if (mitem->GetBitmap().IsOk())
     {
         text = mitem->wxMenuItemBase::GetItemLabel();
-        const wxBitmap *bitmap = &mitem->GetBitmap();
+        const wxBitmap bitmap = mitem->GetBitmap();
 
         // TODO
-        wxUnusedVar(bitmap);
+        wxUnusedVar( &bitmap);
```

Comment:
Minor, but this doesn't really make sense, it should be
```suggestion
        wxUnusedVar(bitmap);
```

Of course, it could probably be even better to stop defining this `bitmap` only not to use it, nobody is clearly ever going to do anything about this "TODO" comment in wxGTK1 code anyhow.